### PR TITLE
[IMP] possibility to set multiple github organizations in GITHUB_ORG arg

### PR DESCRIPTION
**Pull request by legalsylvain**
Hi, Proposing a trivial PR to have the possibility to use a bot for multiple organizations. (I'm currently using it for two organizations).

kind regards.

### CODE REVIEW

**Review by sbidoul — CHANGES_REQUESTED** [review]
I think this will break if GITHUB_ORG is not set.

File: src/oca_github_bot/config.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def func_wrapper(*args, **kwargs):
 GITHUB_SECRET = os.environ.get("GITHUB_SECRET")
 GITHUB_LOGIN = os.environ.get("GITHUB_LOGIN")
 GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
-GITHUB_ORG = os.environ.get("GITHUB_ORG")
+GITHUB_ORG = os.environ.get("GITHUB_ORG", "").split(",")
```
Comment (sbidoul) [inline src/oca_github_bot/config.py:37]: If GITHUB_ORG is not set, this will return `['']` and then the `for org in GITHUB_ORG` wil not do what we expect.

Comment (legalsylvain) [reply src/oca_github_bot/config.py]: Thanks for your review.
Is it now ok ?

Comment (sbidoul) [reply src/oca_github_bot/config.py]: Looks good, thanks!

**Review by sbidoul — APPROVED** [review]

**Pull request merged**